auth/auth.go

```diff
@@ -89,9 +89,20 @@ func Token(ctx context.Context, cfg *config.MountConfig, clientset *kubernetes.C
 				APIVersion: "v1",
 				Name:       cfg.PodInfo.Name,
 				UID:        cfg.PodInfo.UID,
+	resp, err := clientset.CoreV1().
```

Comment:
CI is failing, with:

```
auth/auth.go:92:12: syntax error: unexpected :=, expecting comma or }
auth/auth.go:159:12: no new variables on left side of :=
```

I think there were some lines that you intended to delete above here?